### DISCUSSION TITLE
Information not fiting on screen

### DISCUSSION THREAD

**Discussion by hmontesdeoca91** [svsdval/video2midi#38] [General]
Hi!! you are fantastic, this app is amazing!! I have installed the code on my computer and went into the first steps but once I tried to look for all details it is not able to show color mapping and other details. I shared the picture how it looks and I am not able to drag or move the screen in my desktop to view full details. hope you can help me with some comments. Thanks 
![Screenshot 2023-05-05 135006](https://user-images.githubusercontent.com/132709158/236556025-93734839-d1b3-45e3-b05b-55c16e99c2f9.png)

**Comment by juliuswannick** [comment]
I have the same problem. I am working on a 1600 x 900 screen on a windows 110 PRO Build 19045.2965 version.

**Reply by juliuswannick** [reply]
I realized, that the view has the same size like the video. When i work on a 720p video i see the whole screen.

**Reply by hmontesdeoca91** [reply]
oh so it's the video size I have downloaded, it sounds consistent, I will try today to download another option and come back with the feedback thanks for your support, I checked with F4 and resizing but not working. Come back later

**Reply by hmontesdeoca91** [reply]
Sir, you found the solution now I am able to see it in the correct size. It only works for the 720 option (I am using 4k video downloader) other way in 480p or 1080p is not able to work thanks for your comment and support!!!! I will begin trying how it works and tell you back how is it going, do you have discord or something related? for further discussion?  Regards

 
![Screenshot 2023-05-19 125624](https://github.com/svsdval/video2midi/assets/132709158/7ecb2241-020a-4e17-bd38-aa8115b3acf8)

![Screenshot 2023-05-19 125805](https://github.com/svsdval/video2midi/assets/132709158/47c2ee69-6ac8-4670-bbad-a8049e32a9fb)

**Comment by svsdval** [comment]
Thanks for your help in solving this problem ;)
It looks like it's a bug in the window manager. I'll try to solve this problem on my virtual machine.

**Comment by svsdval** [comment]
Checkout last update ;)

**Reply by hmontesdeoca91** [reply]
Thanks, last one is the 4.7.1? or is a new one?

**Reply by svsdval** [reply]
latest commit or 0.4.7.2

**Comment by hmontesdeoca91** [comment]
new version is working fine with full screen!!! :) 
![Screenshot 2023-05-23 105043](https://github.com/svsdval/video2midi/assets/132709158/c06e1dd7-8fda-4c30-903e-211ab4c5164b)
